components/symbol/tests/unit/symbol.spec.tsx

```diff
@@ -0,0 +1,83 @@
+import { GEL } from '@westpac/core';
+import { render, screen } from '@testing-library/react';
+import wbc from '@westpac/wbc';
+import { overridesTest } from '../../../../helpers/tests/overrides-test.js';
+import { nestingTest } from '../../../../helpers/tests/nesting-test.js';
+import * as symbols from '@westpac/symbol';
+import { Symbol } from '../../src/Symbol.js';
+import { blenderSymbol } from '../../src/overrides/symbol.js';
+
+overridesTest({
+	name: 'symbol',
+	overrides: ['Symbol'],
```

Comment:
Should we include the Svg component in the overrides test?

Reply:
Actually, it through a warning when I added Svg. Is it better to leave out then?
  console.error
    Warning: The tag <metadata> is unrecognized in this browser. If you meant to render a React component, start its name with an uppercase letter.